### PR TITLE
Add warning about postgres' statefulness

### PR DESCRIPTION
**Pull request by G1gg1L3s**
This PR adds warning about the problem described in https://github.com/cossacklabs/acra/pull/537.

Though I'm not sure if it's the right place for such information, but no other options come to my mind.

### CODE REVIEW

File: content/acra/configuring-maintaining/general-configuration/acra-server/encryptor-config.md [hunk 1]
```diff
@@ -350,7 +350,7 @@ Depends on: `data_type`
 
 Group: `encryption`, `searchable encryption`, `masking`
 
-Descryption: specifies which action should be performed in case of a failure of some operation (decryption error, wrong data type, etc.).
+Description: specifies which action should be performed in case of a failure of some operation (decryption error, wrong data type, etc.).
```
Comment (Lagovas) [inline content/acra/configuring-maintaining/general-configuration/acra-server/encryptor-config.md:353]: :eyes:

**Review by Lagovas — APPROVED** [review]
lgtm

**Review by vixentael — APPROVED** [review]

**Comment by vixentael** [comment]
@G1gg1L3s 
On a second thought, I suggest doing the following:

- copy the description of the problem to https://docs.cossacklabs.com/acra/configuring-maintaining/debugging-and-troubleshooting/ page (make new section PostgreSQL and put posgtres related issues there)
- on the Encryptor config page just mention an possible issue and add a link to the Troubleshooting page, exact section

**Comment by G1gg1L3s** [comment]
This is exactly what I was looking for! Thank you :), I will do it.

**Pull request merged**